### DISCUSSION TITLE
关于符号来源

### DISCUSSION THREAD

**Discussion by shitake2333** [Certseeds/wh40k-icon#7] [Announcements]
GW 的 各族 codex 里会有更加详细的符号说明

我在最初制作 nec 和tau 的 icon 的时候，参考来源就使用的是 8 版的 codex

当时计划挨个根据 codex 依次完成所有的  icon 制作。可惜一方面水平的缘故只制作了包含简单符号的 nec 和 tau。另一方面也是因为时间原因放弃。

如果这个项目能继续这个方向完成下去会更棒。

顺便，我已经将我的旧仓库归档。

**Comment by Certseeds** [comment]
> GW 的 各族 codex 里会有更加详细的符号说明
> 
> 我在最初制作 nec 和tau 的 icon 的时候，参考来源就使用的是 8 版的 codex
> 
> 当时计划挨个根据 codex 依次完成所有的 icon 制作。可惜一方面水平的缘故只制作了包含简单符号的 nec 和 tau。另一方面也是因为时间原因放弃。
> 
> 如果这个项目能继续这个方向完成下去会更棒。
> 
> 顺便，我已经将我的旧仓库归档。

bilibili上有up主做了素材库文集, 这个repo就不班门弄斧了, <https://www.bilibili.com/read/readlist/rl881362>
